premailer/tests/test_premailer.py

```diff
@@ -2501,3 +2502,41 @@ def test_remove_unset_properties(self):
         self.assertTrue(p.remove_unset_properties)
         result_html = p.transform()
         compare_html(expect_html, result_html)
+
+    def test_six_colour(self):
+        r = Premailer.six_colour('#cde')
+        e = '#ccddee'
+        self.assertEqual(e, r)
+
+    def test_six_colour_transparent(self):
+        self.assertRaises(TransparentIsNotAColour,
+                          Premailer.six_colour, 'transparent')
+
+    def test_3_digit_colour_expand(self):
+        'Are 3-digit colour values expanded into 6-digits for IBM Notes'
+        html = '''<html>
+  <style>
+    body {background-color: #fe5;}
+    p {background-color: #123456;}
+    h1 {color: #f0df0d;}
+  </style>
+  <body>
+    <h1>Colour test</h1>
+    <p>
+      This is a test of colour handling.
+    </p>
+  </body>
+</html>'''
+        expect_html = """<html>
+  <head>
+  </head>
+  <body style="background-color:#fe5" bgcolor="#ffee55">
+    <h1 style="color:#f0df0d">Colour test</h1>
+    <p style="background-color:#123456" bgcolor="#123456">
+      This is a test of colour handling.
+    </p>
+  </body>
+</html>"""
```

Comment:
Most of the tests in this file try to stick to a convention of indenting the multi-line HTML strings by 4 spaces.